spec/paperclip/processor_spec.rb

```diff
@@ -2,24 +2,24 @@
 
 describe Paperclip::Processor do
   it "instantiates and call #make when sent #make to the class" do
-    processor = mock
-    processor.expects(:make).with()
-    Paperclip::Processor.expects(:new).with(:one, :two, :three).returns(processor)
+    processor = spy
+    expect(processor).to receive(:make)
+    expect(Paperclip::Processor).to receive(:new).with(:one, :two, :three).and_return(processor)
     Paperclip::Processor.make(:one, :two, :three)
   end
 
   context "Calling #convert" do
     it "runs the convert command with Terrapin" do
       Paperclip.options[:log_command] = false
-      Terrapin::CommandLine.expects(:new).with("convert", "stuff", {}).returns(stub(:run))
+      allow(Terrapin::CommandLine).to receive(:new).with("convert", "stuff", {}).and_return(spy(:run))
```

Comment:
should be
`expect(Terrapin::CommandLine).to receive(:new).with("convert", "stuff", {}).and_return(double(run: nil))`